tuneme/templates/polls/includes/poll_free_text_form.html

```diff
@@ -0,0 +1,18 @@
+{% load i18n static poll_votings %}
+
+{% if error_message %}<p><strong>{{ error_message }}</strong></p>{% endif %}
+<form action="{% url 'molo.polls:free_text_vote' question.id %}" method="post">
+  {% csrf_token %}
+  {% if request.user.is_authenticated%}
+    {% can_vote question as user_can_vote %}
+      {% if user_can_vote %}
+        <input type="text" name="vote" id="1" value="" />
```

Comment:
The `id` value is incorrect. It should be `id_vote`. And the name of this input is incorrect, it should be `answer` (change will need to be updated everywhere)